playbooks/roles/mattermost/templates/autolink.json.j2

```diff
@@ -8,10 +8,6 @@
                     {
                         "Pattern": "(?P<project>{{ project }})-(?P<id>\\d{1,6})(?P<comma>[,;]*)",
                         "Template": "[${project}-${id}](https://{{ domain.domain }}/browse/${project}-${id})${comma}"
-                    },
```

Comment:
@SSPJ, I didn't see any issues with this pattern. Is there a reason for removing it?

Reply:
I can't remember why I took it out, sorry.